cpp/lib/include/MarcReader.h

```diff
@@ -0,0 +1,21 @@
+//
+// Created by quboo01 on 16.09.16.
+//
+
+#ifndef UB_TOOLS_MARCREADER_H
+#define UB_TOOLS_MARCREADER_H
+
+#include "MarcRecord.h"
+
+class MarcReader {
+    friend class MarcRecord;
```

Comment:
Having a class w/ only static functions is pointless.  Please put these inside a namespace.  Actually they should probably be in the MarcUtil namespace.